src/service/world-api-adapter.js

```diff
@@ -22,24 +33,49 @@ module.exports = function (config) {
     var store = new StorageFactory({ synchronous: true });
 
     var defaults = {
-
+        /**
+         * For projects that require authentication, pass in the user access token (defaults to empty string). If the user is already logged in to Epicenter, the user access token is already set in a cookie and automatically loaded from there. (See [more background on access tokens](../../../project_access/)).
```

Comment:
I think remove the "(defaults to empty string)" part, doesn't add much more info